test/v1alpha1/revision.go

```diff
@@ -75,10 +75,10 @@ func IsRevisionReady(r *v1alpha1.Revision) (bool, error) {
 	return r.Generation == r.Status.ObservedGeneration && r.Status.IsReady(), nil
 }
 
-// IsRevisionPinned will check if the revision is pinned to a route.
-func IsRevisionPinned(r *v1alpha1.Revision) (bool, error) {
-	_, pinned := r.Annotations[serving.RevisionLastPinnedAnnotationKey]
-	return pinned, nil
+// IsRevisionRoutingActive will check if the revision is actively routing to a route.
+func IsRevisionRoutingActive(r *v1alpha1.Revision) (bool, error) {
+	routingState := r.Labels[serving.RoutingStateLabelKey]
+	return routingState == "active", nil
```

Comment:
not super important to say the least, but just out of interest: `test/v1/revision.go` does `routingState == v1.RoutingStateActive`, any reason not to use the constant here too?

Reply:
didn't want to pull in the v1 package, I find that confusing

Reply:
fair enough (fwiw it's already imported/used in route.go and configuration.go in same package)